nomenclature/definitions/region/subcountries.yaml

```diff
@@ -205,3 +205,32 @@ Italy|Subgrid south:
   note: This region was defined and used in the plan4res public datasets, it is built as aggregation of ehighway2050 regions; ITN+ITS=IT
   ehighway2050: 54_IT + 55_IT + 56_IT + 98_IT
   plan4res: ITS
+  
+# ehighway north sea clusters
+Belgium|North sea: 
```

Comment:
I would think that `North Sea` is more in line with our convention for spelling variables...?

Reply:
@danielhuppmann agree :-)